docs/standard/asynchronous-programming-patterns/toc.yml

```diff
@@ -1,51 +1,52 @@
-- name: Asynchronous Programming Patterns
-  href: index.md
-  items:
-  - name: Task-based Asynchronous Pattern (TAP)
-    href: task-based-asynchronous-pattern-tap.md
+items:
+  - name: Asynchronous Programming Patterns
+    href: index.md
```

Comment:
Suggest splitting index.md out into a standalone child node with the name "Overview".